src/angular-app/bellows/shared/sound-player.component.html

```diff
@@ -2,9 +2,9 @@
     <i class="fa" data-ng-class="$ctrl.iconClass()"></i>
 </a>
 <span class="controls">
-    <span data-ng-if="$ctrl.audioElement.duration" class="audio-progress text-muted">
-        {{$ctrl.currentTime()}} / {{$ctrl.duration()}}
+    <span id="audio-progress" data-ng-if="$ctrl.audioElement.duration" class="audio-progress text-muted">
+        {{$ctrl.currentTime()}} {{$ctrl.duration()}}
     </span>
     <!--suppress HtmlFormInputWithoutLabel -->
-    <input class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" value="0">
+    <input class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" value="0" step="0.0000000000000001">
```

Comment:
:-D I'm very curious as to the why of that level of precision!

Reply:
This is the best way I've found to address a puzzle. The puzzle is that the slider is stopping when it has completed its last full step. Then there is a gap between the slider and the end of the slider, and visually, it appears to have not finished playing. The default "step" size is 1 second, and many smaller sizes are also too big. When the step size approaches 0, the gap becomes invisible. So, a very small step size fixes the appearance of the slider.

Reply:
I recognize that this is kind of a hack, but I have tried many other logical solutions to no avail. If you find something else that clears up this behavior, please send my way

Reply:
do me a favor and drop a comment next to it then explaining the "why" at least for posterity, thanks for being so ingenious! 🧠